src/EventHandler.hs

```diff
@@ -24,6 +25,14 @@ handleKeys (EventKey (SpecialKey KeyRight) Down _ _) game =
 handleKeys (EventKey (SpecialKey KeyRight) Up _ _) game =
       game { paddle = (paddle game) { paddleVel = (0 , 0) }}
 
+-- Moving mouse event , move verticaly the paddle
+handleKeys (EventMotion (x, _)) game
+        = game { paddle = (paddle game) { paddleVel = (x - px , 0)}
+                                        , mouseEvent = True }
```

Comment:
Your indentation is misleading ; it had me believe that `mouseEvent` was a field of `Paddle`.

You could write:

```haskell
-- Moving mouse event , move verticaly the paddle
handleKeys (EventMotion (x, _)) game
        = game { paddle = (paddle game) { paddleVel = (x - px , 0) }
               , mouseEvent = True }
```